internal/resources/ekscluster/data_source_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build ekscluster
```

Comment:
Why this change? Also why was it here in the first place?

Reply:
@asutoshpalai If we have it, test cases in the file are skipped. 
I am not sure why it was added @ramya-bangera  might have some insight